pnpm-lock.yaml

```diff
@@ -753,17 +659,17 @@ packages:
       styled-jsx: 5.0.0
       use-subscription: 1.5.1
     optionalDependencies:
-      '@next/swc-android-arm64': 12.1.0
-      '@next/swc-darwin-arm64': 12.1.0
-      '@next/swc-darwin-x64': 12.1.0
-      '@next/swc-linux-arm-gnueabihf': 12.1.0
-      '@next/swc-linux-arm64-gnu': 12.1.0
-      '@next/swc-linux-arm64-musl': 12.1.0
-      '@next/swc-linux-x64-gnu': 12.1.0
-      '@next/swc-linux-x64-musl': 12.1.0
-      '@next/swc-win32-arm64-msvc': 12.1.0
-      '@next/swc-win32-ia32-msvc': 12.1.0
-      '@next/swc-win32-x64-msvc': 12.1.0
+      '@next/swc-android-arm64': registry.npmmirror.com/@next/swc-android-arm64/12.1.0
```

Comment:
why the pnpm lock file was changed?

Reply:
this is weird, I've added a package to calculate text color based on the background color, but these changes are definitely not anticipated, I'll fix them soon.